src/js/controller/controller.js

```diff
@@ -771,7 +771,12 @@ Object.assign(Controller.prototype, {
 
         function addProgramControllerListeners() {
             _programController
-                .on('all', _trigger, _this)
+                .on('all', (type, event) => {
+                    if (_this._instreamAdapter) {
+                        return;
+                    }
+                    _trigger(type, event);
+                }, _this)
```

Comment:
@pajong did you find any issues today with this change?

@dannyfinks can we get a larger regression test run on this?

One concern I have with DAI is that queueing could cause all of the provider events that fire during instream mode will fire after the adbreak ended.

Reply:
@robwalch will do

Reply:
@robwalch looking at the test results from the smoke test...it seems these changes is causing time events to not be emitted after a VAST Preroll.